addons/weave/template/testgrid/k8s-docker.yaml

```diff
@@ -1,8 +1,31 @@
-- installerSpec:
+- name: "weave latest single node"
+  installerSpec:
     kubernetes:
       version: "latest"
     docker:
       version: "latest"
     weave:
       version: "__testver__"
       s3Override: "__testdist__"
+- name: "weave latest multi node"
+  installerSpec:
+    kubernetes:
+      version: "latest"
+    docker:
+      version: "latest"
+    weave:
+      version: "__testver__"
+      s3Override: "__testdist__"
+  numPrimaryNodes: 1
+  numSecondaryNodes: 2
+- name: "weave 2.6.5 single node"
+  installerSpec:
+    kubernetes:
+      version: "latest"
+    docker:
+      version: "latest"
+    weave:
+      version: "2.6.5"
+      s3Override: "__testdist__"
+  numPrimaryNodes: 1
+  numSecondaryNodes: 2
```

Comment:
```suggestion
```

remove this. you can only test the version its triggered from the workflow

Reply:
Done, thank you. I did notice testgrid using 2.8.1 for this test: `Fetching 3007-c20eb96-weave-2.8.1.tar.gz`